src/lib/constants.js

```diff
@@ -0,0 +1,18 @@
+export const NAVIGATION = [
```

Comment:
I really like how you've separate out all the constants in this file rather than hard coding it within each component file. It makes it much cleaner to read as well as helps to make it more easily editable. This will also probably make it easier to make the whole app translatable in the future.